src/tests/ftest/container/global_handle.py

```diff
@@ -100,44 +91,20 @@ def check_handle(pool_glob_handle, uuidstr, cont_glob_handle, rank):
 
     return
 
-class GlobalHandle(Test):
+class GlobalHandle(TestWithServers):
 
     """
     This class contains tests to verify the ability to share container
     handles amoung processes.
+    :avocado: recursive
     """
 
-    def setUp(self):
-        self.agent_sessions = None
-        # get paths from the build_vars generated by build
-        with open('../../../.build_vars.json') as build_file:
-            self.build_paths = json.load(build_file)
-
-        # setup the DAOS python API
-        self.context = DaosContext(self.build_paths['PREFIX'] + '/lib/')
-
-        server_group = self.params.get("name", '/server_config/',
-                                       'daos_server')
-
-        basepath = os.path.normpath(self.build_paths['PREFIX'] + "/../")
-
-        self.hostlist = self.params.get("test_machines", '/run/hosts/')
-        self.hostfile = write_host_file.write_host_file(self.hostlist,
-                                                        self.workdir)
-
-        self.agent_sessions = AgentUtils.run_agent(basepath, self.hostlist)
-        server_utils.run_server(self.hostfile, server_group, basepath)
-
     def tearDown(self):
         try:
-            os.remove(self.hostfile)
+            super(GlobalHandle, self).tearDown()
```

Comment:
Is the removal of `self.hostfile` especially important for this test? If not I'm fine with the change.

Reply:
`self.hostfile`'s handling is pushed up into the apricot base classes.